charts/openshift-metering/templates/hive/hive-server-statefulset.yaml

```diff
@@ -124,6 +152,67 @@ spec:
 {{- end }}
         resources:
 {{ toYaml .Values.hive.spec.server.resources | indent 10 }}
+{{- if .Values.hive.spec.server.config.metastoreTLS.enabled }}
+      - name: ghostunnel-client
+        image: {{ .Values.__ghostunnel.image.repository }}:{{ .Values.__ghostunnel.image.tag }}
+        imagePullPolicy: {{ .Values.__ghostunnel.image.pullPolicy }}
+        env:
+        - name: GHOSTUNNEL_CLIENT_CA_CRTFILE
+          value: /opt/hive/tls/ca.crt
+        - name: GHOSTUNNEL_CLIENT_KEYSTORE
+          value: /opt/hive/tls/keystore-combined.pem
+        command: ["ghostunnel"]
+        args:
+        - client
+        - --listen
+        - localhost:8443
+        - --target
+        - hive-metastore:8443
```

Comment:
This could potentially go in a `.spec.hive.spec.server.config.metastore` field, if you want to go that route.

Reply:
It's going to be static anyways, and I'm moving away from fields for `hosts` that are mostly unchanging, and using hardcoded values with conditions instead since it makes it easier to write template logic for.

Reply:
Fair enough, this looks fine to me.